ui/pages/swaps/smart-transaction-status/smart-transaction-status.js

```diff
@@ -369,13 +375,16 @@ export default function SmartTransactionStatusPage() {
           <Text
             color={TextColor.textAlternative}
             variant={TextVariant.bodySm}
+            as="h6"
             boxProps={{ marginLeft: 2 }}
           >
             {`~${destinationValue && Number(destinationValue).toFixed(5)} `}
           </Text>
           <Text
             color={TextColor.textAlternative}
-            variant={TextVariant.bodySmBold}
+            variant={TextVariant.bodySm}
```

Comment:
You can still keep `variant={TextVariant.bodySmBold}` here :)

Reply:
Yes, @DDDDDanica ,my bad.
Doing that now.